staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -987,8 +987,6 @@ var (
 	}
 	etpLocalNodePredicates []NodeConditionPredicate = []NodeConditionPredicate{
 		nodeIncludedPredicate,
-		nodeSchedulablePredicate,
```

Comment:
So if I understand, this will immeditately cause a sync when CCM restarts if:

* etp=Local AND
* node is not schedulable or is tainted

Right?  What I mean is that a node is currently NOT in the LB set because of the old predicates, but under the new predicates we would add it to the LB set.  This would cause it start accepting traffic (because the HC changes are not in yet) and then stop serving traffic again when the node was terminated.

Is that a correct assessment?

For eTP=Local that seems like a small risk, but when we want to do this for all services, more important.  I guess that, by then, we'll have the HC changes in place.

Reply:
> This would cause it start accepting traffic (because the HC changes are not in yet)

Yes but the node would also have to be hosting the running pod - otherwise it wouldn't get any traffic sent its way because of the ETP=local HC. Worst case scenario: we re-configure the LB with the not schedulable node, traffic is sent to it because it hosts the currently running pod, but pod is very shortly after stopped. This is not really a problem at all, right? This _might_ be a problem for the readiness predicate though, because maybe the node is not ready because networking is bonkers, but at the same time: it's able to answer to the LB HC, so it's not really that bonkers either 🤷 

> but when we want to do this for all services, more important.

Yes, because we wouldn't be protected by the ETP=local HC. So we'll add a potential "unsafe" (is a "not schedulable node" really "unsafe"?) node back into the LB set, which - worst case scenario - might not be able to forward traffic anywhere...but again, just how bad have things _really_ gone networking wise if the node is still able to send a response to the LB HC?